Philips.CodeAnalysis.DuplicateCodeAnalyzer/AvoidDuplicateStringsAnalyzer.cs

```diff
@@ -0,0 +1,74 @@
+// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
On the performance note, this one was relatively slow at 75ms, but still within reason. It's possibly faster to register per Literal and then save in a Dictionary (a la DuplicateCodeAnalyzer). The dictionary can be segmented per file or per project if desired.

Reply:
Done